source/content/integrated-composer.md

```diff
@@ -214,7 +214,7 @@ Merge the changes manually:
 1. Merge in the upstream changes:
 
    ```bash{promptUser: user}
-   git pull https://github.com/pantheon-upstreams/drupal-project main
+   git remote add [upstream-name] git@github.com:[org]/[project].git
```

Comment:
So, the `remote add` won't actually fetch anything or do the merge.. it just adds a new reference which git won't even do anything with yet--e.g. it won't even notice if the reference is invalid.  To do the merge, the `remote add` would need to be followed by a `fetch` and a `merge`. 

A `pull` is just a `fetch` followed by a `merge`.  I don't think using the `remote add` approach will avoid any permissions conflicts... they will just occur at the `fetch` or `merge` stages.

Since the steps in Solution 1 guided them to get the upstream's git URL, I would simply instruct them to use it as-is.

For the remote-add, fetch, merge change, it would write it as:

```bash{promptUser: user}
git remote add upstream <upstream_url> & git fetch upstream
git merge upstream/main
```

Reply:
thanks, @namespacebrian !
and thanks @erok415 !  does it seem like Brian's suggestion here would help alleviate the issue you encountered, or do you feel like there's another approach that would help future readers?

Reply:
Yes, I think Brian's is more complete as he included the "&& git fetch upstream" reference. I prefer to really spell things out for the users so that they aren't guessing (for novice users). Thank you.